mobile/src/main/java/org/openhab/habdroid/ui/activity/ContentController.kt

```diff
@@ -508,6 +525,34 @@ abstract class ContentController protected constructor(private val activity: Mai
 
     internal abstract fun executeStateUpdate(reason: FragmentUpdateReason)
 
+    override fun updateActionBarState() {
+        val page = temporaryPage
+        val shouldShowActionBar = if (page is AbstractWebViewFragment) {
+            page.wantsActionBar
+        } else {
+            true
+        }
+        val actionBar = activity.supportActionBar
+        if (shouldShowActionBar) {
+            actionBar?.show()
+        } else {
+            actionBar?.hide()
+        }
+        updateContentViewForInsets()
+    }
+
+    private fun updateContentViewForInsets() {
+        val i = insets?.getInsets(WindowInsets.Type.systemBars())
```

Comment:
`WindowInsets.Type` has been added in API 30 (> minSdk).

Reply:
Fixed by using `WindowInsetsCompat`.